### PR TITLE
[#38937] Approve request partner

### PR DESCRIPTION
**Pull request by trunghq-2593**
## Related Tickets
- [#38937](https://edu-redmine.sun-asterisk.vn/issues/38327)
- [#38958](https://edu-redmine.sun-asterisk.vn/issues/38958)
- [#38959](https://edu-redmine.sun-asterisk.vn/issues/38959)

## HOW
- show list hotel of partner for admin
- admin can approved or denied
- pagination
Screenshot from 2021-08-02 11-53-52

### CODE REVIEW

**Comment by trunghq-2593** [comment]
ready

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public function index()
+    {
+        $hotels = Hotel::paginate(config('user.number_of_hotel'));
+
+        return view('cms.pages.admin.index', compact('hotels'));
+    }
+
+    public function upload(Request $request)
+    {
+        $hotel = Hotel::find($request->id);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/AdminController.php:19]: ko tìm thấy hotel thì sao em?

**Comment by trunghq-2593** [comment]
ready

File: resources/views/cms/pages/admin/index.blade.php [hunk 2]
```diff
@@ -5,7 +5,75 @@
...
+                                <th>#</th>
+                                <th>@lang('admin.hotel.name')</th>
+                                <th>@lang('admin.hotel.description')</th>
+                                <th>@lang('admin.hotel.province')</th>
+                                <th>@lang('admin.hotel.status')</th>
+                                <th>@lang('admin.hotel.action')</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @foreach($hotels as $hotel)
```
Comment (ghost) [inline resources/views/cms/pages/admin/index.blade.php]:
```suggestion
                            @foreach ($hotels as $hotel)
```

File: resources/views/cms/pages/admin/index.blade.php [hunk 3]
```diff
@@ -5,7 +5,75 @@
...
+                                <tr>
+                                    <td>#{{ ($hotels->currentPage()-1) * $hotels->perPage() + $loop->iteration }}</td>
+                                    <td>{{ $hotel->name }}</td>
+                                    <td>{{ $hotel->description }}</td>
+                                    <td>{{ \App\Models\Province::find($hotel->province_id)->name }}</td>
```
Comment (ghost) [inline resources/views/cms/pages/admin/index.blade.php]: n+1 query nè :3 tại sao ko load trước ra các province của hotel rùi gọi thôi nhỉ? :3

**Comment by trunghq-2593** [comment]
ready

File: resources/views/cms/pages/admin/index.blade.php (hunk 2 above)
Comment (ghost) [reply resources/views/cms/pages/admin/index.blade.php]: chưa fix nè

**Comment by trunghq-2593** [comment]
ready

**Comment by trunghq-2593** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**